front/app/translations/admin/en.json

```diff
@@ -1367,7 +1367,17 @@
   "app.containers.AdminPage.SettingsPage.fieldTopicTitle": "Tag name",
   "app.containers.AdminPage.SettingsPage.fieldTopicTitleError": "Provide a tag name for all languages",
   "app.containers.AdminPage.SettingsPage.fieldTopicTitleTooltip": "The name you choose for each tag will be visible to the platform users",
+  "app.containers.AdminPage.SettingsPage.fixedRatio": "Fixed-ratio banner",
+  "app.containers.AdminPage.SettingsPage.fixedRatioBannerTooltip": "This banner type works best with images that shouldn’t be cropped, such as images with text, a logo or specific elements that are crucial to your citizens. This banner cannot be combined with an overlay. More info on the recommended image usage can be found on our {link}.",
```

Comment:
Can you replace 
"This banner cannot be combined with an overlay"
 to
 "This banner is replaced with a solid box in the primary colour when users are signed in. You can set this colour in the general settings."?

Reply:
Not a problem to change it, but currently it is possible to combine this banner with an overlay, no? This is why we display the overlay options on the bottom and we show the overlay in the preview. Did I misunderstand a requirement here?